pkg/scheduler/cache/factory.go

```diff
@@ -8,10 +8,7 @@ func RegisterBindMethod(binder Binder) {
 	bindMethodMap = binder
 }
 
+// GetBindMethod get the registered Binder
 func GetBindMethod() Binder {
 	return bindMethodMap
 }
-
```

Comment:
The deletion impact `sc.Binder = GetBindMethod()`  in cache.go

Reply:
I notice that you use another way to initilize the binder.  What is the advantage compared with the init() ? if no more benifit, we'd better keep it as it is :)

Reply:
NewDefaultBinder(old NewBinder) take two params: kubeClient, eventRecorder. It can not be done in init because those params has not be initialized.